src/order-history/service.js

```diff
@@ -1,23 +1,72 @@
 import { getAuthenticatedHttpClient, getAuthenticatedUser } from '@edx/frontend-platform/auth';
-import { getConfig } from '@edx/frontend-platform';
+import { getConfig, initialize, mergeConfig } from '@edx/frontend-platform';
 
 const { ECOMMERCE_BASE_URL } = getConfig();
+
+// Temporary: until we add COMMERCE_COORDINATOR_BASE_URL in frontend-platform, use mergeConfig to join it
+// with the rest of the config items (so we don't need to get it separately from process.env). After we add
+// it to frontend-platform, we'll be able to set it like ECOMMERCE_BASE_URL above, and block below can go away.
+initialize({
+  handlers: {
+    config: () => {
+      mergeConfig({
+        COMMERCE_COORDINATOR_BASE_URL: process.env.COMMERCE_COORDINATOR_BASE_URL || null,
+      });
+    },
+  },
+});
+
 const ECOMMERCE_API_BASE_URL = `${ECOMMERCE_BASE_URL}/api/v2`;
 const ECOMMERCE_RECEIPT_BASE_URL = `${ECOMMERCE_BASE_URL}/checkout/receipt/`;
 
 // eslint-disable-next-line import/prefer-default-export
 export async function getOrders(page = 1, pageSize = 20) {
   const httpClient = getAuthenticatedHttpClient();
   const { username } = getAuthenticatedUser();
+  const { COMMERCE_COORDINATOR_BASE_URL } = getConfig();
+  const orderFetchingUrl = `${COMMERCE_COORDINATOR_BASE_URL}/orders/order_history/`;
 
-  const { data } = await httpClient.get(`${ECOMMERCE_API_BASE_URL}/orders/`, {
+  // [START] TEMPORARY CODE for rollout testing/confirmation===========
+  // Call ecommerce for order info including waffle flag
+  let { data } = await httpClient.get(`${ECOMMERCE_API_BASE_URL}/orders/`, {
     params: {
       username,
       page,
       page_size: pageSize,
     },
   });
 
+  let callCC = false;
+  if (data.count > 0) {
+    callCC = data.results[0].enable_hoist_order_history;
+  }
+  console.log('DKTEMP: enable_hoist_order_history flag is: ', callCC);
+
+  if (callCC) {
+    console.log('DKTEMP: about to call commerce-coordinator');
+    const newData = await httpClient.get(orderFetchingUrl, {
+      params: {
+        username,
+        page,
+        page_size: pageSize,
+      },
+    });
+    data = newData.data;
+    console.log('DKTEMP: CC data.data.results', newData.data.results);
+    console.log('DKTEMP: ecommerce data.results', data.results);
+  }
```

Comment:
This big 'TEMPORARY CODE for rollout testing/confirmation' block is for use to sanity check using the commerce-coordinator endpoint for order history. Therefore I've left a few debugging statements in so our production test can be explicit about where the data came from. Is there a better way to do this, or are these statements okay for this temporary case?

Reply:
As long as we're not logging any PII, and assuming we'll be logged-in in our accounts anyways (so the information will be ours), we should be good!

I would change `DKTEMP` to something theseus/commerce-coordinator related though to be more specific, in case the temporary code is not as temporary as we imagined?

Reply:
Good call- updated the logging prefix to 'REV-2577 LOG' (that'll be the testing ticket), and will confirm whether this log would go to splunk or not.